src/server-extension/resolvers/series.ts

```diff
@@ -20,7 +22,7 @@ enum OrderDirection {
   ASC = 'ASC',
 }
 
-@Resolver()
+@Resolver(of => SeriesEntity)
```

Comment:
👀 what's this?

Reply:
according to [doc](https://typegraphql.com/docs/resolvers.html#field-resolvers), in order to do field resolver, we have to

> First we declare which object type fields we are resolving by providing the type to the @Resolver decorator

From my understanding, it's like passing the SeriesEntity into the field resolver as an argument, so it can use that in the query.

Reply:
Understandable, thanks for the docs